README.md

```diff
@@ -307,8 +307,7 @@ These options are the same regardless of whether you use them with the
 * **splice_read**: deprecated - Does nothing.
 * **splice_write**: deprecated - Does nothing.
 * **splice_move**: deprecated - Does nothing.
-* **allow_other**: deprecated - mergerfs always sets this FUSE option
-  as normal permissions can be used to limit access.
+* **allow_other**: Allow other users to access files. Set by default when running as root.
```

Comment:
The location of the option should probably be moved given it is not really deprecated.

Reply:
I moved this line and updated man